docs/troubleshooting.md

```diff
@@ -0,0 +1,48 @@
+# Troubleshooting
```

Comment:
Add a link to this section from the deployment guide and installation quicksheet.